### DISCUSSION TITLE
Deployment issue - Did you forget to export a function or a server?

### DISCUSSION THREAD

**Discussion by mattbal** [vercel/vercel#11849] [Help] [answered]
### Question

I created a serverless Node.js function and created a cron task to run it once every day.

Code: https://github.com/mattbal/DailyZenQuotes/tree/main/sendmail

When the function runs on Vercel, I get the following error:

```No exports found in module "/var/task/sendmail/api/sendemail.js".
Did you forget to export a function or a server?
Node.js process exited with exit status: 1. The logs above can help with debugging the issue.
INIT_REPORT Init Duration: 5240.70 ms	Phase: init	Status: error	Error Type: Runtime.ExitError
```

I'm confused by this error message. I'm not trying to export a server since I'm not using express. And I'm not really trying to export a function either, I just want the function I've defined to run one time.

Any ideas on how to resolve this error?

**Comment by leerob** [comment, marked as answer]
The code provided doesn't match the [API signature](https://vercel.com/docs/functions/functions-api-reference) for a Vercel Function. You would want something like this for `/api/send.js`:

```js
import nodemailer from 'nodemailer'
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()

export function GET(request: Request) {
  // Add your logic here...
  return new Response('Sent!')
}
```
